bugbear.py

```diff
@@ -136,7 +137,45 @@ def visit(self, node):
 
     def visit_ExceptHandler(self, node):
         if node.type is None:
-            self.errors.append(B001(node.lineno, node.col_offset))
+            self.errors.append(
+                B001(node.lineno, node.col_offset, vars=("bare `except:`",))
+            )
+        elif isinstance(node.type, ast.Tuple):
+            names = [e.id for e in node.type.elts]
+            as_ = " as " + node.name if node.name is not None else ""
+            if len(names) == 0:
+                vs = ("`except (){}:`".format(as_),)
+                self.errors.append(B001(node.lineno, node.col_offset, vars=vs))
+            elif len(names) == 1:
+                self.errors.append(B013(node.lineno, node.col_offset, vars=names))
+            else:
+                # See if any of the given exception names could be removed, e.g. from:
+                #    (MyError, MyError)  # duplicate names
+                #    (MyError, BaseException)  # everything derives from the Base
+                #    (Exception, TypeError)  # builtins where one subclasses another
+                # but note that other cases are impractical to hande from the AST.
+                # We expect this is mostly useful for users who do not have the
+                # builtin exception hierarchy memorised, and include a 'shadowed'
+                # subtype without realising that it's redundant.
+                good = sorted(set(names), key=names.index)
+                if "BaseException" in good:
+                    good = ["BaseException"]
+                for name, other in itertools.permutations(tuple(good), 2):
+                    print(name, other)
```

Comment:
Oops, missed a print 😅

Reply:
We all go this, lucky you noticed it.

I guess we could have CI look for weird output and assert if found.

Reply:
Or flake8-alfred and ban the print function?

Reply:
Ban is such a strong word. Shouldn't you be in bed? :P Hope the fires are not near you mate. 🇦🇺❤️

Reply:
Well... I'm in Canberra, and no family in the way of the fires, which is a relief even with terrible air quality.  On the other hand the place my family camps every summer is in that black patch west of Cooma, the coastal area speaks for itself, and our Prime Minister is beyond hopeless :confused: 

![image](https://user-images.githubusercontent.com/12229877/71754757-bce41080-2edb-11ea-95a0-e7f3519897f6.png)

On the upside, I've removed the stray `print`!